addons/radar/XEH_preInit.sqf

```diff
@@ -80,6 +80,20 @@ if (_acre || _tfar) then {
     GVAR(showSpeaking_replaceIcon) = false;
 };
 
+if (isClass (configfile >> "CfgPatches" >> "ace_medical")) then {
+    [
+        QGVAR(ace_medic)
+        ,"CHECKBOX"
+        ,[localize LSTRING(ace_medic), localize LSTRING(ace_medic_desc)]
```

Comment:
No need for localize, `CBA_fnc_addSetting` checks if it can localize the input.

Could use LLSTRING if you want it localized in this file, yet no need.